db/migrate/20220222150049_create_transactions.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :transactions do |t|
       t.string :action
       t.float :amount
-      t.string :stock
+      t.bigint :stock_id
```

Comment:
Why do you need to modify this migration file? If it is already in the `main` branch better to create a new migration file and apply the needed update to your database structure.

Reply:
Noted, thank you for the feedback. This has been resolved in this commit: https://github.com/jason-nn/batch14-stocks/pull/8/commits/87f6ab797e49c29650885f833e426f8adf7c1524